elfi/methods/bo/acquisition.py

```diff
@@ -405,6 +407,95 @@ def evaluate_gradient(self, theta_new, t=None):
         return gradient
 
 
+class RandMaxVar(MaxVar):
+    r"""The randomised maximum variance acquisition method.
+
+    The next evaluation point is sampled from the variance of
+    the approximate posterior, i.e.:
+    \theta_{t+1} ~ V(p(\theta) * p_t(\theta | X_t)).
```

Comment:
The density can be found in Equation 19 (Marko's old submission). And it can be rearranged as:
<img width="202" alt="screen shot 2017-09-14 at 15 17 46" src="https://user-images.githubusercontent.com/14854408/30429425-0b7f3382-9960-11e7-8856-35692fb61709.png">

Reply:
Yes, but there `p_t` is the BOLFI likelihood i.e. `p_t(X_t | \theta)`. And the whole formulation assumes that.

Reply:
I find it difficult to understand the issue, if we ignore the p(\theta) term, then it is not the MaxVar acquisition. As Marko puts it, "our aim is not to optimise the discrepancy but to minimise our uncertainty in the ABC posterior approximation." This can be found in the paragraph between Equations 18 and 19. As far as my understanding goes, Var(p_t(\theta | X_t)) corresponds to Equation 18 which would be the previously referred discrepancy optimisation.

Reply:
In Bayesian terms prior = p(\theta), likelihood = p_t(X_t | \theta), posterior = p_t(\theta | X_t) and posterior ~ prior * likelihood. You have prior * posterior.

Reply:
The confusion probably comes from the likelihood being determined from the GP's posterior. That's why I'd like to have the BOLFI likelihood eq. here. :)